src/NServiceBus.Core/EndpointCreator.cs

```diff
@@ -158,7 +160,7 @@ void Initialize()
             hostingConfiguration.Container.ConfigureComponent(b => settings.Get<Notifications>(), DependencyLifecycle.SingleInstance);
 
             receiveComponent = ReceiveComponent.Initialize(
-                settings.Get<ReceiveComponent.Configuration>(),
+                settings.Get<ReceiveComponent.Settings>(),
```

Comment:
isn't it a bit weird to pass both settings and configuration to init?

Reply:
Will fix